assemblies/assembly_planning/main.adoc

```diff
@@ -51,16 +51,22 @@ endif::[]
 
 {PRODUCT} {PRODUCT_VERSION} is available for the following application platforms:
 
-* {EAP} 7.1
+* {EAP} 7.2
+* Red Hat JBoss Web Server 5.0
 * Red Hat OpenShift Container Platform
 * Oracle WebLogic Server
 * IBM WebSphere Application Server
 
-=== {EAP} 7.1
-{EAP_LONG} ({EAP}) 7.1 is a certified implementation of the Java Enterprise Edition 7 (Java EE 7) full and web profile specifications. Red Hat JBoss EAP provides preconfigured options for features such as high availability, clustering, messaging, and distributed caching. It also enables users to write, deploy, and run applications using the various APIs and services that Red Hat JBoss EAP provides.
+=== {EAP} 7.2
+{EAP_LONG} ({EAP}) 7.2 is a certified implementation of the Java Enterprise Edition 7 (Java EE 7) full and web profile specifications. Red Hat JBoss EAP provides preconfigured options for features such as high availability, clustering, messaging, and distributed caching. It also enables users to write, deploy, and run applications using the various APIs and services that Red Hat JBoss EAP provides.
 
 You can install both {CENTRAL} and {KIE_SERVER} in a single {EAP} instance. However, in production environments you should install them in separate instances.
 
+=== Red Hat JBoss Web Server 5.0
+Red Hat JBoss Web Server is an enterprise-ready web server designed for medium and large applications, based on Tomcat. Red Hat JBoss Web Server provides organizations with a single deployment platform for Java Server Pages (JSP) and Java Servlet technologies, PHP, and CGI.
+
+You can install {KIE_SERVER} and the {HEADLESS_CONTROLLER} on Red Hat Web Server 5.0.
```

Comment:
@emmurphy1 - I'm sorry, but can you describe in great detail what the {HEADLESS_CONTROLLER} is? TEEEEEEEEEEE HEEEEEEEEEEEEEEE